check_plugin_viewer_fields.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
hello, could you please validate on https://www.shellcheck.net/ (there are lot of warnings)

Reply:
thx, I addressed the warnings in it